lms/djangoapps/teams/management/commands/reindex_course_team.py

```diff
@@ -52,16 +49,21 @@ def handle(self, *args, **options):
         # happen anywhere else that I can't figure out how to avoid it :(
         from ...search_indexes import CourseTeamIndexer
 
-        if len(args) == 0 and not options.get('all', False):
-            raise CommandError(u"reindex_course_team requires one or more arguments: <course_team_id>")
-        elif not settings.FEATURES.get('ENABLE_TEAMS', False):
-            raise CommandError(u"ENABLE_TEAMS must be enabled to use course team indexing")
+        if options['all']:
```

Comment:
Nit: `add_mutually_exclusive_group` can be used here as well (perhaps) - no big deal:
https://docs.python.org/2/library/argparse.html#argparse.ArgumentParser.add_mutually_exclusive_group

Reply:
See my code comment where the arguments are defined---mutually exclusive groups in argparse only work with optional args.